client/src/components/Home/PartnersSection/PartnersSection.css

```diff
@@ -0,0 +1,84 @@
+.partners-section {
+      font-family: 'Raleway', sans-serif;
+      color: white;
+      border: 1px solid #EE2A7B;
+      /* box-sizing: border-box; */
+      border-radius: 23px;
+      width: 85vw;
+      padding-bottom: 50px;
+}
+
+.partners-title {
+      color: #EE2A7B;
+      font-size: 7vh;
+      text-align: center;
+      margin-bottom: 10vh;
+      margin-top: 10vh;
+}
+
+.partners-logo {
+      width: 30vw;
+      border-radius: 20px;
+}
+
+.partners-col-1 {
+      text-align: center;
+      margin-bottom: 40px;
+}
+
+.partners-col-2 {
+      text-align: center;
+      margin-bottom: 40px;
+}
+@media(max-width:500px){
+.partners-section{
+border:none;
+}
+
+}

```

Comment:
Everything looks nice and responsive too 🙌  just fix the indentation here with same tab size for consistency throughout the code and you are ready to go!! 💯

Reply:
Done :-)